chia/server/start_wallet.py

```diff
@@ -29,14 +29,18 @@ def service_kwargs_for_wallet(
 ) -> Dict:
     overrides = config["network_overrides"]["constants"][config["selected_network"]]
     updated_constants = consensus_constants.replace_str_to_bytes(**overrides)
+    full_node_config = load_config(DEFAULT_ROOT_PATH, "config.yaml", "full_node")
```

Comment:
This should be in the wallet config not full node config

Reply:
Hmm I thought of that originally, but then wondered if we're fine with that value existing in two places in config.yaml?

Reply:
We generally deal with options in multiple places but wanting an (optional) single point of truth with yaml anchors.  See `self_hostname` and the like.